### PR TITLE
RSPEC files for #4207 (S6354)

### PR DESCRIPTION
**Pull request by andrei-epure-sonarsource**
cc @Corniel 

To generate this I had to update the rspec.ps1 script - see #5172

### CODE REVIEW

File: analyzers/rspec/cs/S6354_c#.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<p>One of the principles of a unit test is that it must have full control of the system under test. This is problematic when production code includes
+calls to static methods, and date/time functions are usually provided by system libraries as static methods.</p>
```
Comment (Corniel) [inline analyzers/rspec/cs/S6354_c#.html:2]: which behavior can not be changed/controlled

**Review by Corniel — CHANGES_REQUESTED** [review]
It has to be made explicit why statics functions might be an issue here.

**Comment by pavel-mikula-sonarsource** [comment]
Is this PR needed? Changed files on a branch should do the trick.

**Comment by Corniel** [comment]
> Is this PR needed? Changed files on a branch should do the trick.

It's both fine by me.

**Comment by andrei-epure-sonarsource** [comment]
I've updated https://github.com/SonarSource/rspec/pull/285 and pushed the new version.
Indeed, this PR is not needed, the branch is fine.

**Comment by andrei-epure-sonarsource** [comment]
(don't delete the branch please)

**Pull request closed without merging**